vscode-dotnet-runtime-library/src/test/unit/DotnetCoreAcquisitionWorker.test.ts

```diff
@@ -1,216 +1,215 @@
-            await acquisitionWorker.acquireRuntime('1.0');
-        }
-        // We should only actually Acquire once
-        const events = eventStream.events.filter(event => event instanceof DotnetAcquisitionStarted);
-        assert.equal(events.length, 1);
-    });
-
-    test('Error is Redirected on Acquisition Failure', async () => {
-        const context = new MockExtensionContext();
-        const eventStream = new MockEventStream();
-        const acquisitionWorker = new DotnetCoreAcquisitionWorker({
-            storagePath: '',
-            extensionState: context,
-            eventStream,
-            acquisitionInvoker: new RejectingAcquisitionInvoker(eventStream),
-            installationValidator: new MockInstallationValidator(eventStream),
-            timeoutValue: 10,
-            installDirectoryProvider: new RuntimeInstallationDirectoryProvider(''),
-        });
-
-        return assert.isRejected(acquisitionWorker.acquireRuntime('1.0'), '.NET Acquisition Failed: Installation failed: Rejecting message');
-    });
-
-    test('Repeated SDK Acquisition', async () => {
-        const [acquisitionWorker, eventStream, context] = getTestAcquisitionWorker(false);
-        for (let i = 0; i < 3; i ++) {
-            await acquisitionWorker.acquireSDK('5.0');
-        }
-        // We should only actually Acquire once
-        const events = eventStream.events.filter(event => event instanceof DotnetAcquisitionStarted);
-        assert.equal(events.length, 1);
-    });
-});
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+import * as chai from 'chai';
+import * as chaiAsPromised from 'chai-as-promised';
+import * as os from 'os';
+import * as path from 'path';
+import { DotnetCoreAcquisitionWorker } from '../../Acquisition/DotnetCoreAcquisitionWorker';
+import { RuntimeInstallationDirectoryProvider } from '../../Acquisition/RuntimeInstallationDirectoryProvider';
+import { SdkInstallationDirectoryProvider } from '../../Acquisition/SdkInstallationDirectoryProvider';
+import {
+    DotnetAcquisitionCompleted,
+    DotnetAcquisitionStarted,
+    DotnetAcquisitionStatusResolved,
+    DotnetAcquisitionStatusUndefined,
+    DotnetUninstallAllCompleted,
+    DotnetUninstallAllStarted,
+    TestAcquireCalled,
+} from '../../EventStream/EventStreamEvents';
+import { EventType } from '../../EventStream/EventType';
+import {
+    MockEventStream,
+    MockExtensionContext,
+    MockInstallationValidator,
+    NoInstallAcquisitionInvoker,
+    RejectingAcquisitionInvoker,
+} from '../mocks/MockObjects';
+const assert = chai.assert;
+chai.use(chaiAsPromised);
+
+suite('DotnetCoreAcquisitionWorker Unit Tests', function() {
```

Comment:
It's hard to tell with the diff, but it looks like there aren't any tests for the new behavior, is that intentional?

Reply:
This file did not change whatsoever, sorry for the weird diff issue!

The acquisition worker code is tested in the 'e-2-e' test by the sdk extension tests so I didnt test it here. That would not cover any additional code and is a significantly more time-expensive test that most,  yet it also has no added code coverage.